bin/GameIndex.dbf

```diff
@@ -4653,7 +4656,7 @@ Serial = SCUS-97159
 Name   = Ico [Demo]
 Region = NTSC-U
 Compat = 5
-eeClampMode = 2 // Otherwise freezes in various spots, check full intro
+eeClampMode = 2 // Otherwise freezes in various spots, check full intro.
 vuClampMode = 1	// Otherwise camera gets stuck off the player in various spots.
```

Comment:
You should replace the tab by a space between 1 and // here.

Reply:
I will fix this in the other PR.